metrics_api/lib/opentelemetry/metrics/meter.rb

```diff
@@ -37,6 +38,10 @@ def create_histogram(name, unit: nil, description: nil)
         create_instrument(:histogram, name, unit, description, nil) { HISTOGRAM }
       end
 
+      def create_gauge(name, unit: nil, description: nil)
+        create_instrument(:gauge, name, unit, description, nil) { GAUGE }
+      end
```

Comment:
Since the other methods are getting improved documentation in https://github.com/open-telemetry/opentelemetry-ruby/pull/1720, could you update the documentation for this method to match the style in that PR?

Reply:
Updated